src/auro-datepicker.js

```diff
@@ -57,9 +56,12 @@ class AuroDatePicker extends LitElement {
 
     // Lion Calendar options
     this.centralDate = new Date(); /* default to today */ // eslint-disable-line no-inline-comments
-
     this.maxDate = undefined;
     this.minDate = undefined;
+
+    /**
+     * @private
+     */
```

Comment:
nit-pick: it's helpful to have a description, even if it's private.

Reply:
We don't have descriptions for any properties unless they are public for any component that I am aware of.

Reply:
Look at line 289 of this file? 

``` 
  /**
   * Binds all behavior needed to the dropdown after rendering.
   * @private
   * @returns {void}
   */
```

Reply:
@blackfalcon That's a method, not a property. No properties in the constructor have descriptions unless they are public. This is true of all of our components.